runtime.md

```diff
@@ -122,8 +137,3 @@ This specification does not mandate the name of this JSON file.
 See the specification of the `config.json` file for the definition of these fields.
 The stopping, or exiting, of these secondary process MUST have no effect on the state of the container.
 In other words, a container (and its PID 1 process) MUST NOT be stopped due to the exiting of a secondary process.
-
-## Hooks
-
-Many of the operations specified in this specification have "hooks" that allow for additional actions to be taken before or after each operation.
```

Comment:
On Fri, Apr 22, 2016 at 01:25:23PM -0700, Vish Kannan wrote:
“Let me try to summarize the plan for supporting post-stop hooks:”

The current spec pretty clear that post-stop hooks (step 8) happen
after container teardown (step 7).  So the “namespace-pinning and
post-stop hooks” discussion seems to be orthogonal to this PR.

For your sub-container example, I don't think the order/parent-ness
matters much.  You have the pinning container joining the working
container, but you could also have the working container join the
pinning container.  In the latter case, you're basically doing work
via exec, and the sub-container looks like #391.

“1. What parts of `A`'s environment is sharable with `B`?”

There are many reasonable answers to this question.  If I was writing
this up, I'd probably have the parent only create the sandbox it
wanted to pin.  THe child would join, create the un-pinned sandbox,
execute, and die.  Then the child sandbox gets reaped, you pull your
data out of the pinned sandbox, and kill the parent.

[1]: https://github.com/opencontainers/runtime-spec/blob/v0.5.0/runtime.md#lifecycle